### PR TITLE
Fix and Improve markdown markup in 07.spreading-props-dom.md

### PR DESCRIPTION
**Pull request by Ch-Valentine**

### CODE REVIEW

File: anti-patterns/07.spreading-props-dom.md [hunk 1]
```diff
@@ -1,12 +1,11 @@
 # Spreading props on DOM elements
 When we spread props we run into the risk of adding unknown HTML attributes, which is a bad practice.
 
-###Reference:
-- React Design Patterns and best practices by Michele Bertoli
-
+### Reference:
+[React Design Patterns and best practices by Michele Bertoli.](https://github.com/MicheleBertoli/react-design-patterns-and-best-practices)
```
Comment (Reda-bx) [inline anti-patterns/07.spreading-props-dom.md]: add `-` please to make list

Comment (Ch-Valentine) [inline anti-patterns/07.spreading-props-dom.md]: @Reda-bx to the Reference ?

Comment (Reda-bx) [inline anti-patterns/07.spreading-props-dom.md]: Yes

**Comment by Ch-Valentine** [comment]
@Reda-bx, done.

**Pull request merged**